.ci/install_kata_image.sh

```diff
@@ -144,26 +133,7 @@ if [ -f "${cidir}/${ARCH}/lib_kata_image_${ARCH}.sh" ]; then
 fi
 
 main() {
-	if [ x"${TEST_INITRD}" == x"yes" ]; then
-		build_image
-	else
-		# If installing packaged image from OBS fails,
-		# then build and install it from sources.
-		rc=0
-		install_packaged_image || rc=1
```

Comment:
this `install_packaged_image` function will no longer be used, can you remove it?